doc/west/index.rst

```diff
@@ -1,14 +1,75 @@
 .. _west:
 
-West (Experimental)
-###################
+West
+####
 
-The Zephyr project is developing a swiss-army knife command line tool
+The Zephyr project maintains a swiss-army knife command line tool
 named ``west``. (Zephyr is an English name for the Greek god of the
 west wind.)
 
-West is developed in its own `repository on GitHub`_. A copy is
-currently maintained in Zephyr's :file:`scripts/meta/west` directory.
+West is developed in its own `repository on GitHub`_.
+
+West is used to obtain the source code for the Zephyr project and can also be
+used to build, debug and flash applications.
+
+.. _west-struct:
+
+Structure
+*********
+
+West is split into two distinct parts that work together to provide the
+full set of functionality offered by the tool:
+
+* Bootstrapper: Installed using ``pip``, implements ``west init``
+* Installation: Installed using ``west init``, implements all other commands
+
+Additional information about the two parts can be found below.
+
+Bootstrapper
+============
+
+The bootstrapper module is distributed using PyPi (i.e. :file:`pip`) and it's
+placed in the user's ``PATH`` becoming the only entry point to west.
+It implements a single command: ``west init``. This command needs to be run
+first in order to use the rest of functionality included in ``west``, since
+``west init`` will do the following:
+
+* Clone West itself in a :file:`west/west` folder
+* Clone the manifest in a :file:`west/manifest` folder (additional information
+  on the manifest can be found in the :ref:`west-multi-repo` section)
+
+Once ``west init`` has been run, the bootstrapper will delegate the handling of
+any commands different from ``init`` to the cloned West installation.
+
+This means that there is a single bootstrapper instance installed at any time
+(unless you use virtual environments), which can then be used to initialize as
+many installations as needed.
+
+.. _west-struct-installation:
+
+Installation
+============
+
+A West installation is the result of running ``west init`` on a folder. As
+described on the previous section, an installation always includes a special
+:file:`west/` folder that contains a clone of both west itself and the manifest.
+Additionally, and once ``west clone`` has been run, the installation will also
+contain a clone of any repositories listed in the manifest. The directory tree
+layout of an installation using the default Zephyr manifest therefore looks
+like this:
```

Comment:
Worth mentioning the config file here too

Reply:
done

Reply:
Here's an updated tree as well:

```
├── west
│   ├── config
│   ├── manifest
│   └── west
└── zephyr
```

Reply:
I purposefully avoided having `config` in that tree. The reason is that I only wanted to show folders and not files, since I describe it as "directory tree", and I did not want people to think that `config` was a folder. Any suggestions?